File: git_repo_searcher_kotlin/app/src/main/java/com/example/gitreposearcher/model/RepoInfoModel.kt
package com.example.gitreposearcher.model

data class RepoInfoModel(
    val ownerLogin: String,
    val name: String,
    val description: String?,
    val starsCount: Int,
    val forksCount: Int,
    val issueCount: Int,
    val pullRequestsCount: Int,
    val readME: String?
)

File: git_repo_searcher_flutter/lib/model/repo_info_model.dart
import 'package:built_value/built_value.dart';

part 'repo_info_model.g.dart';

abstract class RepoInfoModel
    implements Built<RepoInfoModel, RepoInfoModelBuilder> {
  RepoInfoModel._();

  String get ownerLogin;

  String get name;

  @nullable
  String get description;

  int get starsCount;

  int get forksCount;

  int get issueCount;

  int get pullRequestCount;

  @nullable
  String get readMe;

  factory RepoInfoModel([void Function(RepoInfoModelBuilder) updates]) =
      _$RepoInfoModel;
}
